content/integrate/write-behind/data-transformation/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: Data transformation
+aliases: null
+alwaysopen: false
+categories:
+- docs
+- integrate
+- rs
+- rdi
+description: null
+group: di
+hideListLinks: false
+linkTitle: Data transformation
+summary: Redis Data Integration keeps Redis in sync with the primary database in near
+  real time.
+type: integration
+weight: 30
+---
+
+The key functionality that Write-behind performs is mapping the data coming from [Debezium Server](https://debezium.io/documentation/reference/stable/operations/debezium-server.html) (representing a Source Database row data or row state change) into a Redis key with a value of [Hash]({{< relref "/develop/data-types/hashes" >}}) or [JSON]({{< relref "/develop/data-types/json/" >}}).
+There are two types of data transformations in Write-behind:
+
+1. By default, each source row is converted into one [Hash]({{< relref "/develop/data-types/hashes" >}}) or one JSON key in Redis.
```

Comment:
Probably don't need this URL here as the previous paragraph has it.

Reply:
Done. As with the emphasis, I'll check for things like this throughout.